### ISSUE TITLE
Common gas vs LNG

### ISSUE THREAD

**Issue by phiphou** [electricitymaps/electricitymaps-contrib#4823]
Hi,

Do you take in account LNG in gas datas ? I've been told LNG is up to 2.5 times more CO2 because of shipping and liquefaction.

**Comment by VIKTORVAV99** [comment]
I don't think we currently do but I have talked with @pierresegonne with it and we both agreed it would be a nice addition if we can find the data and the time to create it. Unfortunately, everyone is quite busy at the moment but hopefully it's something we can implement in the future.

**Comment by asfi30** [comment]
Hi,

I doubt you can estimate LNG only.  Usually at the point of electricity generation it is a mix of CNG,LNG, Stored Gas, Gas from production. For instance in Bulgaria we have NO LNG terminals, but we have virtual swap of CNG and LNG gas with Greece that has LNG terminals. Biogas from recycling could be CNG as well. I mean CNG is  mainly Methane (CH4) and LNG is a bit purified and liquefied Methane as well. When you start taking into account transport, gasification, degasification, swap, electricity production transmition, distribution, losses then you have to calculate the so called Primary energy. I am giving you just an example. One cubic meter (1m3) of CNG has 10KWh if you burn it at oven at home. If you make Electricity those 10KWh are maximum 3.6KWh electricity plus 5KWh heat dissipated and 1.4KWh overall losses in CHP plant. But all the Co2eq emissions are given to the electricity.

**Comment by VIKTORVAV99** [comment]
The idea I discussed with Pierre was to flow trace all gas similar to how we do it for the electricity flow, then create a life cycle emission model that comprises of one static part that account for all infrastructure and one dynamic part that accounts for the fuel (gas).

As you probably can tell already this would be a big project as we would have to find new data sources, build new parsers, create new flow tracing models, new gas grid zones, etc.

While we both agreed that this would generate the most accurate gas emissions the shear amount of work needed don't make it a viable option right now. But hopefully it's something we can do in the future.
